### PR TITLE
[MM-20309] production public key

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR will update the production public key

#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/MM-20309

### CODE REVIEW

File: tests/development-private-key.asc [hunk 1]
```diff
@@ -0,0 +1,82 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```
Comment (ali-farooq0) [inline tests/development-private-key.asc]: any reason we want to keep the private key? is it just for testing?

Comment (iomodo) [reply tests/development-private-key.asc]: Yes, for testing

**Comment by ali-farooq0** [comment]
@iomodo will leave it to you to merge this when other changes are ready to go 👍

**Pull request merged**